File: CodeWars/JavaScript/sumOfNumbers.js
function getSum( a,b )
{
  //sum = 0 , biasakan
  let sum = 0;

///// Solusi Yang Pertama 
// kondisi apabila a < b , maka a diloop sampai dgn b
//   if (a<b){
//     for (i = a ; i <= b ; i++){
//       sum += i; 
//     }
//   } else if (a>b){
//     for (i = b ; i <= a ; i++){
//       sum += i; 
//     }
//   } else if (a ==b) {
//     sum = a;
//   }
  
//   return sum;
// }

//// Solusi yang kedua 
if (a >b) {
  [a,b] = [b,a];
}

while (a<=b){
  sum += a;
  a++;
}

return sum;

}

console.log(getSum(1,2))

File: CodeWars/Golang/sumOfNumbers.go
package main

import "fmt"

func getSum(a, b int) int {

	sum := 0

	//cara pertama yang kompleks
	if a < b {
		for i := a; i <= b; i++ {
			sum += i
		}
	} else if b < a {
		for i := b; i <= a; i++ {
			sum += i
		}
	} else if a == b {
		sum = a
	}
	return sum
}

func main() {
	fmt.Println(getSum(-1, 2))
}
